dev/archery/archery/cli.py

```diff
@@ -692,6 +692,8 @@ def _set_default(opt, default):
               help="Seed for PRNG when generating test data")
 @click.option('--with-cpp', type=bool, default=False,
               help='Include C++ in integration tests')
+@click.option('--with-csharp', type=bool, default=False,
+              help='Include CSharp in integration tests')
```

Comment:
Should it be spelled "C#" in this message?

Reply:
Yes, good call.